Formula/openrct2.rb

```diff
@@ -17,17 +17,16 @@ class Openrct2 < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "nlohmann-json" => :build
   depends_on "pkg-config" => :build
   depends_on "duktape"
-  depends_on "freetype" # for sdl2_ttf
+  depends_on "freetype"
   depends_on "icu4c"
   depends_on "libpng"
   depends_on "libzip"
   depends_on macos: :mojave # `error: call to unavailable member function 'value': introduced in macOS 10.14`
-  depends_on "nlohmann-json"
   depends_on "openssl@1.1"
   depends_on "sdl2"
-  depends_on "sdl2_ttf"
```

Comment:
This dependency was removed a while ago: https://github.com/OpenRCT2/OpenRCT2/commit/2c07a556963022653b3147db8b941655b5e20476

Nice catch.